entry_types/paged/packages/pageflow-paged/src/editor/views/configurationEditors/groups/pageTransitions.js

```diff
@@ -17,6 +17,6 @@ ConfigurationEditorTabView.groups.define('page_transitions', function(options) {
   }
 
   this.input(options.propertyName || 'page_transition',
-             SelectInputView,
+              ThumbnailInputView,
```

Comment:
The change is only supposed to take effect in Pageflow Scrolled. Page transitions in Pageflow Paged work differently and do not need to be considered here.

Reply:
I have rever that. Thanks